src/lib.rs

```diff
@@ -21,13 +21,14 @@ pub struct Font<'a> {
 	offset: usize
 }
 
+/// The requested glyph with character code could not be found.
 #[derive(Clone, Copy, Error)]
-#[error("no such glyph")]
-pub struct NoSuchGlyph(());
+#[error("no such glyph {}", .0)]
+pub struct NoSuchGlyph(char);
 
 impl Debug for NoSuchGlyph {
 	fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
-		f.debug_tuple("NoSuchGlyph").finish()
+		write!(f, "NoSuchGlyph")
```

Comment:
I don't like this change. (a) because it is a debug impl and should therefore use the debug calls, and (b) because the struct now has useful data that should not only be in the display impl but also the debug impl. Therefore we can just use the derive macro.

Previously the field only served the purpose of the existence of a private field so that only this library could instantiate the error type, so I manually implemented the debug impl to hide that implementation detail.